src/tests/Interop/DllImportSearchPaths/DllImportSearchPathsTest.cs

```diff
@@ -95,14 +114,28 @@ public static void AssemblyDirectory_SearchFlags_WithDependency_Found()
 
 public class NativeLibraryPInvoke
 {
+    internal const string CopyName = $"{NativeLibraryToLoad.Name}-copy";
+    internal const string DefaultFlagsName = $"{NativeLibraryToLoad.Name}-default-flags";
+
     public static int Sum(int a, int b)
-    {
-        return NativeSum(a, b);
-    }
+        => NativeSum(a, b);
+
+    public static int Sum_Copy(int a, int b)
+        => NativeSum_Copy(a, b);
+
+    public static int Sum_DefaultFlags(int a, int b)
+        => NativeSum_DefaultFlags(a, b);
```

Comment:
It might be worth marking these with `[MethodImpl(MethodImplOptions.NoInlining)]` to provide a consistent test environment - no inlining in one flavor vs the other.